### PR TITLE
Safelisting in webmail

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1b9b75d6cb/how-to/)

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- **Time needed:** Approximately 5 minutes
+- **Tools required:**  apps.rackspace.com access
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+### What does Safelisting accomplish?
+
+In short, **Safelisting** an address creates a rule that moves any message flagged as spam from the address you **Safelisted** to the **Inbox** so that you do not miss the message. If you need to **Blacklist** a sender, see [Blacklist addresses, domains and IPs in Rackspace Email webmail](/how-to/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail).
+
+**Warning:** Safelisting DOES NOT prevent a sender from being flagged as spam or from being blocked.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Remove capitalization on "Does NOT".

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- **Applies to:** User
+- **Difficulty:** Easy
+- **Time needed:** Approximately 5 minutes
+- **Tools required:**  apps.rackspace.com access
+
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+### What does Safelisting accomplish?
+
+In short, **Safelisting** an address creates a rule that moves any message flagged as spam from the address you **Safelisted** to the **Inbox** so that you do not miss the message. If you need to **Blacklist** a sender, see [Blacklist addresses, domains and IPs in Rackspace Email webmail](/how-to/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail).
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Remove "In short..."

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    <img src="{% asset_path rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail/safelist_settings.png %}"/>
+
+3. Click **Spam Settings** on the left hand side of the pop-up box.
+
+4. Select the second tab, which is labeled **Safelist**.
+
+    <img src="{% asset_path rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail/settings_safelist.png %}"/>
+
+5. Click **Add** in the box that is titled **Safelisted Domains & Email Addresses**.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: "In the Safelisted Domains and Email Addresses box, click **Add**."

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+6. When you are prompted with a box that is titled **Add Safelisted Domain or Email Address**, enter the domain or email address you wish to Safelist list in the field under **Enter a domain or email address**, and then click **Add**.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: "When prompted to **Add Safelisted Domain or Email Address**, enter..."

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    <img src="{% asset_path rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail/safelist_domain.png %}"/>
+
+7. Confirm that the address is now listed in the box under **Safelisted Domains & Email Addresses**, and finally click **Save**.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Remove "...finally..."

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+### Add a Safelisted IP address
+
+If you need to Safelist and IP, follow steps 1-4 above, and continue here.
+
+1. Click **Add** in the box titled **Safelisted IP Addresses**.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: In the Safelisted IP Addresses box, click **Add**.

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+2. When you are prompted with a box that is titled **Add Safelisted IP Address**, enter the IP address that you wish to Safelist in the field under **Enter an IP address(or range)**, and then click **Add**.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Same note as line 47.

File: content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+    <img src="{% asset_path rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail/safelist_ip.png %}"/>
+
+3. Confirm that the address is now listed in the box under **Safelisted Domains & Email Addresses**, and finally click **Save**.
```
Comment (DonNateR) [inline content/rackspace-email/safelist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Remove "...finally...".

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a6b2d92e8a/how-to/)

**Comment by catlook** [comment]
@stephamon please review, thx

**Comment by stephamon** [comment]
Hey @catlook here are my suggestions. Let me know if you have any questions!

[Safelist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.docx](https://github.com/rackerlabs/rackspace-how-to/files/1467934/Safelist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.docx)
[Safelist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1467935/Safelist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.pdf)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5392900880/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**